demos/face_recognition_demo/python/face_recognition_demo.py

```diff
@@ -0,0 +1,313 @@
+    general.add_argument('-o', '--output',
+                         help='Optional. Name of output to save.')
+    general.add_argument('-limit', '--output_limit', default=1000, type=int,
+                         help='Optional. Number of frames to store in output. '
+                              'If 0 is set, all frames are stored.')
+    general.add_argument('--output_resolution', default=None, type=resolution,
+                         help='Optional. Specify the maximum output window resolution '
+                              'in (width x height) format. Example: 1280x720. '
+                              'Input frame size used by default.')
+    general.add_argument('--no_show', action='store_true',
+                         help="Optional. Don't show output.")
+    general.add_argument('-cw', '--crop_width', default=0, type=int,
+                         help='Optional. Crop the input stream to this width. '
+                              'Both -cw and -ch parameters should be specified '
+                              'to use crop.')
+    general.add_argument('-ch', '--crop_height', default=0, type=int,
+                         help='Optional. Crop the input stream to this height. '
+                              'Both -cw and -ch parameters should be specified '
+                              'to use crop.')
+    general.add_argument('--match_algo', default='HUNGARIAN', choices=('HUNGARIAN', 'MIN_DIST'),
+                         help='Optional. Algorithm for face matching. Default: HUNGARIAN.')
+    general.add_argument('-u', '--utilization_monitors', default='', type=str,
+                         help='Optional. List of monitors to show initially.')
+
+    gallery = parser.add_argument_group('Faces database')
+    gallery.add_argument('-fg', type=Path, required=True,
+                         help='Required. Path to the face images directory.')
+    gallery.add_argument('--run_detector', action='store_true',
+                         help='Optional. Use Face Detection model to find faces '
+                              'on the face images, otherwise use full images.')
+    gallery.add_argument('--allow_grow', action='store_true',
+                         help='Optional. Allow to grow faces gallery and to dump on disk. '
+                               'Available only if --no_show option is off.')
+
+    models = parser.add_argument_group('Models')
+    models.add_argument('-m_fd', type=Path, required=True,
+                        help='Required. Path to an .xml file with Face Detection model.')
+    models.add_argument('-m_lm', type=Path, required=True,
+                        help='Required. Path to an .xml file with Facial Landmarks Detection model.')
+    models.add_argument('-m_reid', type=Path, required=True,
+                        help='Required. Path to an .xml file with Face Reidentification model.')
+    models.add_argument('-fd_iw', '--fd_input_width', default=0, type=int,
```

Comment:
I suggest to use one argument instead of two `-fd_iw`, `-fd_ih`:
```
'-fd_resize', type=resolution, default=None
```

Reply:
Updated.